gallery_dl/extractor/cyberdrop.py

```diff
@@ -0,0 +1,43 @@
+import re
+import base64
+
+from .common import Extractor, Message
+
+
+class CyberdropAlbumExtractor(Extractor):
+    pattern = r"(?:https?://)?(?:www\.)?cyberdrop\.me/a/([^/]+)/?"
+    category = "cyberdrop"
+    subcategory = "album"
+    directory_fmt = ("{category}", "{album}")
+    root = "https://cyberdrop.me"
+
+    def __init__(self, match):
+        Extractor.__init__(self, match)
+        self.album_id = match.group(1)
+        self.album_url = self.root + "/a/" + self.album_id
+
+    def items(self):
+        initial_page_response = self.request(self.album_url)
+
+        albumNameRegex = r"name: '([^']+)'"
+        albumName = re.findall(albumNameRegex, initial_page_response.text)[0]
+
+        fileListRegex = r"fl:.'([^']+)'"
+        results = re.findall(fileListRegex, initial_page_response.text)
+        if (len(results)) != 1:
+            raise RuntimeError("could not find file list in html")
+
+        fileList = [base64.b64decode(s.encode('ascii')).decode('utf8')
```

Comment:
I don't think those arguments for `encode` and `decode` are necessary.
The default (`utf-8`) should work fine.